solr/solr-ref-guide/src/monitoring-solr-with-prometheus-and-grafana.adoc

```diff
@@ -108,6 +108,38 @@ The number of seconds between collecting metrics from Solr. The `solr-exporter`
 
 The Solr's metrics exposed by `solr-exporter` can be seen at: `\http://localhost:9983/solr/admin/metrics`.
 
+=== Getting metrics from a secure Solr(Cloud)
+
+Your Solr(Cloud) might be secured by measures described in <<securing-solr.adoc#securing-solr,Securing Solr>>. The security configuration can be injected into `solr-exporter` using environment variables in a fashion similar to other clients using <<using-solrj.adoc#using-solrj,SolrJ>>. This is possible because the main script picks up two external environment variables and passes them on to the Java process:
+
+* `JAVA_OPTS` allows to add extra JVM options
+* `CLASSPATH_PREFIX` allows to add extra libraries
+
+Example for a SolrCloud instance secured by
+
+* <<basic-authentication-plugin.adoc#basic-authentication-plugin,Basic Authentication>>
```

Comment:
Instead of a bullet list, just make it a sentence : "Example for a SolrCloud instance secured by both Basic Authentication, SSL and ZooKeeper Access Control:"

Reply:
fixed